src/app/settings.py

```diff
@@ -303,8 +303,8 @@ def show_toolbar(request):
 
 CACHES = {
     'default': {
-        'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
-        'LOCATION': 'locmem',
+        'BACKEND': 'django.core.cache.backends.db.DatabaseCache',
```

Comment:
Any reason not to keep it in memory? Too large?

Are there any other places in the code where we were/are using cache that need to get cleaned up or refactored for this?

Reply:
Doesn't support cross processor